### PR TITLE
k*.py: normalize docs

### PR DESCRIPTION
**Pull request by russoz**
##### SUMMARY

Normalize doc blocks.

##### ISSUE TYPE

- Docs Pull Request

##### COMPONENT NAME

plugins/modules/kdeconfig.py
plugins/modules/kernel_blacklist.py
plugins/modules/keycloak_authentication.py
plugins/modules/keycloak_authentication_required_actions.py
plugins/modules/keycloak_authz_authorization_scope.py
plugins/modules/keycloak_authz_custom_policy.py
plugins/modules/keycloak_authz_permission.py
plugins/modules/keycloak_authz_permission_info.py
plugins/modules/keycloak_client.py
plugins/modules/keycloak_client_rolemapping.py
plugins/modules/keycloak_client_rolescope.py
plugins/modules/keycloak_clientscope.py
plugins/modules/keycloak_clientscope_type.py
plugins/modules/keycloak_clientsecret_info.py
plugins/modules/keycloak_clientsecret_regenerate.py
plugins/modules/keycloak_clienttemplate.py
plugins/modules/keycloak_component.py
plugins/modules/keycloak_component_info.py
plugins/modules/keycloak_group.py
plugins/modules/keycloak_identity_provider.py
plugins/modules/keycloak_realm.py
plugins/modules/keycloak_realm_info.py
plugins/modules/keycloak_realm_key.py
plugins/modules/keycloak_realm_keys_metadata_info.py
plugins/modules/keycloak_realm_rolemapping.py
plugins/modules/keycloak_role.py
plugins/modules/keycloak_user.py
plugins/modules/keycloak_user_federation.py
plugins/modules/keycloak_user_rolemapping.py
plugins/modules/keycloak_userprofile.py
plugins/modules/keyring.py
plugins/modules/keyring_info.py
plugins/modules/kibana_plugin.py
plugins/modules/krb_ticket.py

### CODE REVIEW

**Review by ahussey-redhat — APPROVED** [review]
Keyring modules look good to me

File: plugins/modules/keycloak_realm_keys_metadata_info.py [hunk 1]
```diff
@@ -9,42 +9,39 @@
 
 __metaclass__ = type
 
-DOCUMENTATION = """
----
+DOCUMENTATION = r"""
 module: keycloak_realm_keys_metadata_info
 
-short_description: Allows obtaining Keycloak realm keys metadata via Keycloak API
+short_description: Allows obtaining Keycloak realm keys metadata vusingia Keycloak API
```
Comment (felixfontein) [inline plugins/modules/keycloak_realm_keys_metadata_info.py:15]:
```suggestion
short_description: Allows obtaining Keycloak realm keys metadata using Keycloak API
```

File: plugins/modules/kibana_plugin.py [hunk 2]
```diff
@@ -11,71 +11,70 @@
...
-      - "Timeout setting: 30s, 1m, 1h etc."
-      default: 1m
-      type: str
-    plugin_bin:
-      description:
+      - For local file, prefix its absolute path with file://.
+    type: str
+  timeout:
+    description:
+      - 'Timeout setting: 30s, 1m, 1h and so on'
```
Comment (felixfontein) [inline plugins/modules/kibana_plugin.py:46]:
```suggestion
      - 'Timeout setting: 30s, 1m, 1h and so on.'
```

**Review by felixfontein — APPROVED** [review]

**Comment by felixfontein** [comment]
@russoz thanks for your contribution!
@ahussey-redhat thanks for reviewing!

**Pull request merged**